packages/perseus/src/widgets/table/score-table.ts

```diff
@@ -9,7 +9,7 @@ import type {PerseusStrings} from "../../strings";
 import type {PerseusScore} from "../../types";
 import type {
     PerseusTableRubric,
-    PerseusTableUserInput,
+    PerseusTableUserInput, UserInput,
```

Comment:
This doesn't seem right, does it? I wonder why we're not getting a linting error or something..